runtime/port/unix/j9sysinfo.c

```diff
@@ -838,24 +838,31 @@ getS390Description(struct J9PortLibrary *portLibrary, J9ProcessorDesc *desc)
 	if (J9_ARE_NO_BITS_SET(*(int*) 200, S390_STFLE_BIT)) {
 		return -1;
 	}
-#elif defined(LINUX) && !defined(J9ZTPF) /* LINUX S390*/
+#elif defined(J9ZTPF)  /* ZTPF */
+	// First turn off all features
+	unsigned long auxvFeatures = 0;
```

Comment:
The values can be assigned directly.  No need for the initial 0 assignment first.

Reply:
Done.